prebid_cache_client/client.go

```diff
@@ -70,14 +70,7 @@ type clientImpl struct {
 }
 
 func (c *clientImpl) GetExtCacheData() (string, string) {
-	path := c.externalCachePath
-	if path == "/" {
```

Comment:
I think it makes sense to leave in the `path == "/"` check as returning an empty string instead of empty path should be good and seems preferred as a way to communicate "there is no path". Then we should revert the Index check such as if the index has length >0 and doesn't start with a "/" to prepend it so we enforce it always being present to avoid breaking UC.js.